docs/metadata.md

```diff
@@ -0,0 +1,73 @@
+
+## Dispute
+
+Structure from the graph
+
+```javascript
+type Dispute @entity {
+  id: ID!
+  ...
+  subject: Arbitrable!
+  metadata: String!
+  ...
+}
+```
+
+```javascript
+type Arbitrable @entity {
+  id: ID!
+  disputes: [Dispute!] @derivedFrom(field: "subject")
+}
+```
+
+In the frontend we try to parse the `metadata` as a json object.
+
+### metadata - Json object
+
+We are going to try to find the following keys:
```

Comment:
We should update this: if the `metadata` is a json, we now only expect:

```js
1. `description`
2. `metadata`
```

Where `metadata` contains:

```
1. `agreementText` (required)
2. `description`
3. `defendant`
4. `plaintiff`
```